websubmit/Bibtex.py

```diff
@@ -102,7 +102,9 @@ def process_references(references, output_format):
     nsmsg = '*** Non-standard form, no INSPIRE lookup performed ***'
     for ref in references:
         index = None
-        if re.search(r'.*\:\d{4}\w\w\w?', ref):
+        if re.search(r'\d{4}[\w.&]{15}', ref):
```

Comment:
I think the pattern should be anchored `r'^\d{4}[\w.&]{15}$'` 
it avoids matching something in a longer string and avoids unnecessary search

this doesn't necessarily apply to the other patterns, which are less specific,
e.g. for new arXiv identifiers 
`r'\d{4}\.\d{4,5}'` also allows for prefix, like `\cite{arxiv:1234.5678}`, so not having that anchored is good. there may also be version extensions (shouldn't, and probably won't match in lookup unless Inspire metadata also has that version extension

Reply:
for example a garbled citation (say an accidental double paste)
`\cite{2019ApJ...884..168G019ApJ...884..168G2}`
will match and lead to output

```
<div class="notfound">\bibitem{2019ApJ...884..168G019ApJ...884..168G2}
        *** Not Found with lookup: '035:"2019ApJ...884..168G019ApJ...884..168G2"'
</div>
```

while it should say
```
<div class="nonstandard">\bibitem{2019ApJ...884..168G019ApJ...884..168G2}
        *** Non-standard form, no INSPIRE lookup performed ***
</div>
```